test/infrastructure/docker/docker/types/node.go

```diff
@@ -34,13 +34,15 @@ type Node struct {
 	Name        string
 	ClusterRole string
 	InternalIP  string
+	Image       string
 	Commander   *containerCmder
 }
 
 // NewNode returns a Node with defaults.
-func NewNode(name, role string) *Node {
+func NewNode(name, image, role string) *Node {
```

Comment:
It is used to feed the image the container was created with up the stack. For example, when listing containers using `docker ps` it's helpful to add the image name to the `Node` so that we can introspect what version of K8s is running on that container.

Reply:
thanks for the clarification